Makefile

```diff
@@ -14,6 +14,9 @@ GO_BUILD_ARGS=CGO_ENABLED=0 GO111MODULE=on
 build:
 	$(GO_BUILD_ARGS) GOOS=windows go build -o wmcb.exe  $(MAIN_PACKAGE)
 
+run:
```

Comment:
`run` what? Are you assuming that ansible and other packages are installed as part of `build` then? I wouldn't make changes in the make file unless, I am 100% of how the flow is going to look like.

Reply:
Run the WMCB. I'll gladly remove it from the makefile.